README.md

```diff
@@ -314,6 +316,16 @@ Other optional arguments to `nlsolve`, available for all algorithms, are:
 * `extended_trace`: should additifonal algorithm internals be added to the state
   trace? Default: `false`.
 
+## Fixed Points 
+
+There is a `fixedpoint()` wrapper around `nlsolve()` which maps an input function `F(x)` to `G(x) = F(x) - x`, and likewise for the in-place. This allows convenient solution of fixed-point problems, e.g. of the kind commonly encountered in computational economics. Some notes:
+
+* The default method is `:anderson` with `m = 0`, which corresponds to simple iteration on a contraction mapping.
+* Autodifferentiation is supported; e.g. `fixedpoint(f!, init_x; method = :newtonm. autodiff = :true`).
+* Tolerances and iteration bounds can be set exactly as in `nlsolve()`, since this function is a wrapper, e.g. `fixedpoint(f, init_x; inplace = false, iterations = 500, ...)`. 
+
+**Note:** If you are supplying your own derivative, it is currently not automatically mapped, so it must correspond with `F(x) = x`. 
```

Comment:
"it is currently not automatically mapped, so it must correspond with `F(x) = x" is not super clear here, what does this actually mean, and how should the user work around this?

Reply:
The idea is that we currently don't map `J(x) -> J(x) - I` in the way that we map `f(x) -> f(x) - x`, in order to turn finding a 0 into finding a fixed-point. Will make that clearer.

Reply:
Ah, okay, I get it. Why?

Reply:
It's on the list for things to implement. Some parts of this API are still in flux (like broadcasting on `UniformScaling`, so I'd rather wait until this is settled.

Reply:
Done.